koswat/strategies/order_strategy/order_strategy.py

```diff
@@ -72,7 +63,9 @@ def get_strategy_reinforcements(
     def apply_strategy(
         self, strategy_input: StrategyInput
     ) -> list[StrategyLocationReinforcement]:
-        _reinforcement_order = self.get_default_order_for_reinforcements()
+        _reinforcement_order = OrderStrategyReinforcements.from_strategy_input(
+            strategy_input
+        ).strategy_reinforcements
```

Comment:
It does not make sense to me to use a class method for this when the only usecase for this class is delivering the `strategy_reinforcements`.